<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExercisesDialog</class>
 <widget class="QDialog" name="ExercisesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des exercices</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/images/exercise-icon.png</normaloff>:/images/exercise-icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Exercices</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="pListView" name="lstExercises"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Informations</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0" colspan="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Nom :</string>
           </property>
           <property name="buddy">
            <cstring>leName</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Type :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="rbRepetition">
           <property name="text">
            <string>Répétition</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>bgType</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="rbDuration">
           <property name="text">
            <string>Durée</string>
           </property>
           <attribute name="buttonGroup">
            <string>bgType</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QCheckBox" name="cbUseWeight">
           <property name="text">
            <string>Utilisation de poids</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Difficulté :</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="2">
          <widget class="QComboBox" name="cbDifficulty"/>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLineEdit" name="leName"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Groupes musculaires</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Groupe 1 : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbGroup1"/>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="chkGroup2">
           <property name="text">
            <string>Groupe 2 :</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbGroup2">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="chkGroup3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Groupe 3 :</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cbGroup3">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Description</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="pteDescription">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Actions</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="margin">
          <number>6</number>
         </property>
         <property name="spacing">
          <number>6</number>
         </property>
         <item row="0" column="0" rowspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btnDelete">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/images/delete-icon.png</normaloff>:/images/delete-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnModify">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Modifier</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/images/save-icon.gif</normaloff>:/images/save-icon.gif</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="btnAdd">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
           <property name="icon">
            <iconset resource="ressources.qrc">
             <normaloff>:/images/add-icon.png</normaloff>:/images/add-icon.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pListView</class>
   <extends>QListView</extends>
   <header>widgets/pView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkGroup2</sender>
   <signal>clicked(bool)</signal>
   <receiver>cbGroup2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGroup3</sender>
   <signal>clicked(bool)</signal>
   <receiver>cbGroup3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkGroup2</sender>
   <signal>clicked(bool)</signal>
   <receiver>chkGroup3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExercisesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="bgType"/>
 </buttongroups>
</ui>
